fastestimator/summary/system.py

```diff
@@ -259,7 +263,8 @@ def save_state(self, save_dir: str) -> None:
             'ds': {
                 mode: {key: value.__getstate__()
                        for key, value in ds.items() if hasattr(value, '__getstate__')}
-                for mode, ds in self.pipeline.data.items()
+                for mode,
```

Comment:
formatting still?

Reply:
I updated the style but it still formats the same.